intel_extension_for_transformers/llm/runtime/graph/models/gptneox/gptneox_utils.cpp

```diff
@@ -45,6 +45,7 @@ void model_load_internal(const std::string& fname, model_archs arch, model_conte
   std::unique_ptr<GPTNEOX> ms(new GPTNEOX());
   ms->init(fname.c_str(), lctx, n_gpu_layers, use_mmap, use_mlock, vocab_only);
   ms->load(lctx, progress_callback, progress_callback_user_data);
+  lctx.support_jblas_kv = true;
```

Comment:
when can we set it as default true? @DDEle

Reply:
Most of models now support fused-attention now: https://github.com/intel/intel-extension-for-transformers/blob/main/intel_extension_for_transformers/llm/runtime/graph/docs/fused_attention.md#supported-models

IMHO, it is better to let model owners to explicitly confirms that they have implemented fused-attention support for their models.

Reply:
@a32543254 @zhentaoyu @zhenwei-intel @CeciliaWwq  any comments?

Reply:
OK for me